Documentation/policy.rst

```diff
@@ -174,7 +174,19 @@ The following example builds on top of the previous one but requires that *all*
 endpoints which carry the label ``env=prod`` require the consumer to also carry
 the label ``env=prod`` in order for access to be granted:
 
-.. literalinclude:: ../examples/policies/requires.json
+.. only:: html
+
+   .. tabs::
+     .. group-tab:: JSON
+
+        .. literalinclude:: ../examples/policies/requires.json
+     .. group-tab:: YAML
+
+        .. literalinclude:: ../examples/policies/requires.yaml
+
+.. only:: epub or latex
```

Comment:
IMO yaml looks better on books no? json is too verbose

Reply:
We can change it to YAML. Also if you want can change the default on the web to show YAML first. Is probably enough to change the order for it to take effect.

There is a separate issue for runtime tests. Have assigned it to me. Will try to follow up this weekend or next week.

Thanks.